debug.cabal

```diff
@@ -43,7 +43,7 @@ library
         aeson,
         containers,
         ghc-prim,
-        Hoed >= 0.4.1 && < 0.5,
+        Hoed >= 0.5,
```

Comment:
Is it worth saying < 0.6 given how tightly coupled we are? Your call.

Reply:
Err no. Hoed 0.6 might or might not break debug so attempting to guess is futile. I'm a big supporter of upper bounds to enable automatic dependency solving, but I am against using them for preemptive guesswork.

Reply:
Fine by me. I generally agree with that approach.